### PR TITLE
Lionel

### PR DESCRIPTION
**Pull request by spykelionel**
# ft/user - added user/id route
# ft/post - added post/id route

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   "author": "",
   "license": "MIT",
   "scripts": {
-    "start": "node src/index.js",
+    "start": "nodemon src/index.js",
```
Comment (mernxl) [inline package.json]: Why did you make this change, You could easily run that as 

```
npm run dev
```

File: src/index.js [hunk 2]
```diff
@@ -34,6 +34,31 @@ app.get('/users', (req, res) => {
   res.send(users);
 });
 
+// will get a particular user
+app.get('/users/:id', (req, res) => {
+  let id = req.params.id;
+  let user = users.find(user => user.id == id);
+  if(typeof(user) === "undefined"){
```
Comment (mernxl) [inline src/index.js]: You don't need to check the typeof, just doing the following works

```js
...
let user = users.find(user => user.id == id);
if (user) {
...
```

**Review by mernxl — COMMENTED** [review]
Apart from the few, your approach is fine.

If you want your work merged, then do the changes and ping me.

**Comment by mernxl** [comment]
Good Job.

**Comment by spykelionel** [comment]
OK I get it.

**Comment by spykelionel** [comment]
thanks

**Pull request merged**